boskos/janitor/deployment.yaml

```diff
@@ -17,6 +17,7 @@ spec:
       - name: boskos-janitor
         image: gcr.io/k8s-testimages/janitor:v20180725-c421ed4f9
         args:
+        - --boskos-url=http://boskos.test-pods.svc.cluster.local
```

Comment:
Here and below, we should include the trailing dot for FQDN:
```
http://boskos.test-pods.svc.cluster.local.
```

Reply:
d'oh, done!